cli/src/main/java/denominator/cli/Denominator.java

```diff
@@ -309,6 +323,43 @@ public void run() {
       }
     }
 
+    private Function<Object, Object> attemptToTransformCredentials = new Function<Object, Object>() {
```

Comment:
make things here static unless you are referencing an instance variable. also make the function final. I like the intent of "attemptToTransformCredentials" except that it doesn't say much about what it is doing. perhaps con 

perhaps `decodeAnyPems()` as that implies that a pem may or may not be present.